parsepatch/patch.py

```diff
@@ -64,11 +65,11 @@ def lines_chunk(it):
             add_lines_for_new=add_lines_for_new,
         )
         p.changeset = chgset
-        return p.parse()
+        return p.parse(get_hunks=get_hunks)
```

Comment:
nit: to be coherent, you should put the option in the Patch ctor

Reply:
I have addressed this by adding get_hunks to the Patch constructor.